docs/UserGuide.md

```diff
@@ -3,41 +3,47 @@ layout: page
 title: User Guide
 ---
 
-WorkBook (WB) is a **desktop app for CS/tech students who are applying for internships to manage their internship applications, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, WB can get your internship management tasks done faster than traditional GUI apps.
-
-
+<div class="toc-no-bullet-points">
 * Table of Contents
 {:toc}
+</div>
+
+--------------------------------------------------------------------------------------------------------------------
+## 1. Introduction to WorkBook
+
+WorkBook is an internship application tracker that helps Computing students **prepare sufficiently** for their upcoming interviews to **secure** that internship.
```

Comment:
are we no longer mentioning "tracking internship applications"?

Reply:
Internship application tracker I suppose? I'll bold it.

Reply:
oops I completely missed it due to the contrast of the bolded words